integration_tests/snaps/ros2/snap/snapcraft.yaml

```diff
@@ -0,0 +1,21 @@
+name: ros2-example
+version: '0.1'
+summary: ROS2 Example
+description: |
+  A ROS2 workspace containing a talker and a listener.
+
+grade: stable
+confinement: strict
+
+apps:
+  launch-project:
+    command: launch $(ros2 pkg prefix listener_py)/share/listener_py/launch/talk_and_listen.py
+    plugs: [network]
+    environment:
+      # Required to get interleaved stdout, see
+      # https://discourse.ros.org/t/ros2-launch-not-interleaving-stdout
+      PYTHONUNBUFFERED: 1
```

Comment:
How about setting this in the plugin's env so it doesn't have to be repeated in every snap?

Reply:
Because I currently believe it to be a hackaround as a result of using Python, whose support is new. It was not requires in ROS1, and I expect ROS2's launch will eventually take care of it as well. If not then we can revisit.